c2cgeoform/templates/site/index.pt

```diff
@@ -17,7 +17,8 @@
       <script type="text/javascript">
         $('#grid').bootgrid({
             ajax: true,
-            url: '${request.route_url('grid', schema=schema.name)}',
+            multiSort: true,
```

Comment:
here you activate multisort on client side, but this seems not supported on server side for the moment

Reply:
Multisort done on server side in a2bcdf5